.rubocop.yml

```diff
@@ -1,4 +1,14 @@
+inherit_from:
+  - https://raw.githubusercontent.com/alphagov/govuk-lint/master/configs/rubocop/all.yml
+
 Style/ConditionalAssignment:
   Enabled: false
 Style/StringLiterals:
   Enabled: false
+
+AllCops:
+  Include: 
+    - 'app/**/*.rb'
+    - 'lib/**/*.rb'
+  Exclude:
+    - 'bin/**'
```

Comment:
Did you also want to exclude spec?

Reply:
I wanted only include stuff, but for some reason it was still including the bin folder in addition to app and bin, so I had to exclude it here 🤷‍♂ ...